app/Locale/Slovak.php

```diff
@@ -303,7 +302,6 @@ public function get_locale(): array
 			'SUCCESS' => 'OK',
 			'RETRY' => 'Opakovať',
 
-			'SETTINGS_WARNING' => 'Zmena rozšírených nastavení môže mať negatívny vplyv na stabilitu, bezpečnosť a rýchlosť tejto aplikácie. Upravujte len to, o čom presne viete, čo robíte.',
```

Comment:
This one is actually used in `view.js` by `view.full_settings.content.init`. We must not delete it.